endpoints/openrtb2/auction.go

```diff
@@ -1567,3 +1584,30 @@ func getAccountID(pub *openrtb2.Publisher) string {
 	}
 	return metrics.PublisherUnknown
 }
+
+func storedRequestErrorChecker(err error, requestJson []byte, storedRequests map[string]json.RawMessage, storedBidRequestId string) ([]byte, []error) {
+	if err != nil {
+		hasErr, Err := getJsonSyntaxError(requestJson)
+		if hasErr {
+			err = fmt.Errorf("Invalid JSON in Incoming Request: %s", Err)
+		} else {
+			hasErr, Err = getJsonSyntaxError(storedRequests[storedBidRequestId])
+			if hasErr {
+				fmt.Println(err)
+				err = fmt.Errorf("Invalid JSON in Stored Request with ID %s: %s", storedBidRequestId, Err)
+				err = fmt.Errorf("ext.prebid.storedrequest.id refers to Stored Request %s which contains Invalid JSON: %s", storedBidRequestId, Err)
```

Comment:
Which error message do you want to use here? The first one is currently overwritten by the second. The formatting of the second error is more Go-like, as it doesn't start with an upper case letter.